.travis.yml

```diff
@@ -2,6 +2,9 @@ language: python
 python:
   - "2.7"
   - "3.3"
+  - "3.4"
+  - "3.5"
+  - "pypy3"
```

Comment:
I am not aware that Travis supports that. PyPy 3 supports only the 3.2
specification right now with the important difference that it supports
unicode strings (so not supporting 3.2 but pypy3 is actually a useful
thing.)
On 16 Nov 2015 12:47 a.m., "Todd Wolfson" <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/twolfson/restructuredtext-lint/pull/24#discussion_r44880196>
> :
>
> > @@ -2,6 +2,9 @@ language: python
> >  python:
> >    - "2.7"
> >    - "3.3"
> > +  - "3.4"
> > +  - "3.5"
> > +  - "pypy3"
>
> Judging by the name pypy3 is a major version. Should we lock down to
> minor version as we do with CPython? (e.g. 3.4)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/twolfson/restructuredtext-lint/pull/24/files#r44880196>
> .
>